apps/api/src/app/rate-limiting/usecases/get-api-rate-limit-maximum/get-api-rate-limit-maximum.usecase.ts

```diff
@@ -1,21 +1,21 @@
 import { Injectable, InternalServerErrorException, Logger } from '@nestjs/common';
```

Comment:
This is a rename from `get-api-rate-limit` -> `get-api-rate-limit-maximum` to better reflect the fact that a maximum allowed rate limit is being fetched for the given environment entity.